### ISSUE TITLE
Rendering of COLR/CPAL color font sometimes fails

### ISSUE THREAD

**Issue by emojifreak** [latex3/luaotfload#95]
Rendering of color fonts in COLR/CPAL format sometimes fails. For example

```
\documentclass{minimal}
\usepackage{luaotfload}
\usepackage{bxcoloremoji}

\begin{document}
\font\fontemoji={name:TwemojiMozilla.ttf:+colr;+ccmp}
U+1F494 {\fontemoji 💔} broken heart\\
For comparison, \coloremoji{💔} which is also Twemoji font.
\end{document}
```

generates the following PDF by lualatex:

[luaotf-bug.pdf](https://github.com/latex3/luaotfload/files/3586120/luaotf-bug.pdf)

Twemoji font is version 0.4 downloaded from https://github.com/mozilla/twemoji-colr/releases

**Comment by emojifreak** [comment]
I observed a similar issue in ConTeXt MkIV and reported it to http://tracker.luatex.org/view.php?id=1012

**Comment by u-fischer** [comment]
The bxcoloremoji contains simply a lot of pdf and png pictures. It is quite unclear if it shows the glyph as it is meant to be. Which other proof have you that the heart should be colored in both parts?

**Comment by emojifreak** [comment]
@u-fischer Thank you for your interest.

> Which other proof have you that the heart should be colored in both parts?

I show three evidences:
(1) Look at
https://github.com/twitter/twemoji/blob/master/assets/72x72/1f494.png

(2) Harflatex 0.4.1 produces a PDF in the same design as (1)

(3) The PDF in bxcoloremoji is produced by inkscape.

The correctness of the current behavior of ConTeXt&LuaTeX&LuaLaTeX logically implies that (A) TTF conversion from SVG to TTF by Mozilla is incorrect, (B) Harflatex 0.4.1 is incorrect and (C) SVG-to-PDF conversion by inkscape is incorrect. It seems unlikely to me that all of Mozilla and harflatex and inkscape are simultaneously incorrect to produce the broken heart in the same incorrect way. How do you think❓🤔

By the way, you can find all the differences between bxcoloremoji version of twemoji and LuaLaTeX version at
https://github.com/emojifreak/All-Emoji-by-HarfTeX-and-LuaTeX-ja/blob/master/emoji-full7-no-harftex.pdf

**Comment by emojifreak** [comment]
The related issue was fixed in ConTeXt http://tracker.luatex.org/view.php?id=1012#c1716

**Comment by emojifreak** [comment]
This problem seems fixed in luaotfload 3.1 and there seems no reason to keep this open. Thank you very much for your work.
